### DISCUSSION TITLE
Installing Server 2008 Updates on Vista

### DISCUSSION THREAD

**Discussion by Gravarty** [LegacyUpdate/LegacyUpdate#46] [Ideas]
It would be great to also download and install Windows Server 2008 updates for Vista using this site.

**Comment by kirb** [comment]
Server 2008 should be supported - anything labelled as Vista should equally apply to Server 2008, since the desktop and server variants share almost all updates in common from 2008 onwards. Have you tried it, and if so are you having problems with it? I can set up a VM again to give it a test.

**Reply by Gravarty** [reply]
Oh, sorry. By that I meant installing Server 2008 Updates that were released after Windows Vista lost support.
Server 2008 updates adds a newer version of DirectX, Timezone fixes, Spectrum/Meltdown patch and some things that are required for the Extended Kernel to work. You can install all these updates on any version of Windows Vista without any problems, but manually installing all these updates released after 2017 is reallly tedious :D

**Reply by PeterStrick** [reply]
> Oh, sorry. By that I meant installing Server 2008 Updates that were released after Windows Vista lost support. Server 2008 updates adds a newer version of DirectX, Timezone fixes, Spectrum/Meltdown patch and some things that are required for the Extended Kernel to work. You can install all these updates on any version of Windows Vista without any problems, but manually installing all these updates released after 2017 is reallly tedious :D

Kind of like the POSReady 2009 Patch for XP. Would be nice if the Legacy Update Page had an option to also install Server 2008 R1 Updates on Vista

**Comment by Octal450** [comment]
@kirb Any update on this?
